cirq-core/cirq/sim/classical_simulator.py

```diff
@@ -0,0 +1,80 @@
+from typing import Dict
+from collections import defaultdict
+
+import cirq
+import numpy as np 
+
+
+class ClassicalSimulator(cirq.SimulatesSamples):
+
+
+    """
+    very basic simulator that only accepts Xgates and CNot gates and return a 3d Numpy array
+
+    Run a simulation, mimicking quantum hardware.
```

Comment:
This seems misleading, since it is running only classical operations.  Maybe we could adjust this description.